src/libraries/Microsoft.PowerFx.Connectors/Execution/FormulaValueSerializer.cs

```diff
@@ -193,6 +199,64 @@ private async Task WritePropertyAsync(string propertyName, ISwaggerSchema proper
                 return;
             }
 
+            if (_schemaLessBody && propertySchema?.Type == null)
+            {
+                if (propertyName != null)
+                {
+                    WritePropertyName(propertyName);
+                }
+
+                if (fv is NumberValue numberValue)
+                {
+                    WriteNumberValue(numberValue.Value);
+                }
+                else if (fv is DecimalValue decimalValue)
+                {
+                    WriteDecimalValue(decimalValue.Value);
+                }
+                else if (fv is BooleanValue booleanValue)
+                {
+                    WriteBooleanValue(booleanValue.Value);
+                }
+                else if (fv is StringValue stringValue)
+                {
+                    WriteStringValue(stringValue.Value);
+                }
+                else if (fv is DateTimeValue dtv)
+                {                    
+                    WriteDateTimeValue(_utcConverter.ToUTC(dtv));                                        
+                }
+                else if (fv is DateValue dv)
+                {
+                    WriteDateValue(dv.GetConvertedValue(null));
+                }                
+                else if (fv is BlobValue bv)
+                {
+                    await WriteBlobValueAsync(bv).ConfigureAwait(false);
+                }
+                else if (fv is OptionSetValue optionSetValue)
+                {
+                    WriteStringValue(optionSetValue.Option);
+                }
+                else if (fv is RecordValue recordValue)
+                {
+                    StartObject();
+
+                    foreach (NamedValue field in recordValue.Fields)
+                    {
+                        await WritePropertyAsync(field.Name, null, field.Value).ConfigureAwait(false);
+                    }
+
+                    EndObject();
+                }
+                else
+                {
+                    throw new NotImplementedException($"Unsupported type {fv.GetType().FullName} for schemaless body");
+                }
+

```

Comment:
Can we share this code with somewhere else? 
It seems especially easy to miss a case.